File: Week 2 C/interest.c
#include <stdio.h>

float calculate_interest(float rate, float principle);

int main(void){
  float rate;
  float principle;
  printf("Enter the interest rate as a percentage: ");
  scanf("%f", &rate);
  printf("Enter the principle amount of your loan: ");
  scanf("%f", &principle);

  printf("Your simple interest amount is %.2f \n", calculate_interest(rate,principle));
  return 0;
}

float calculate_interest(float rate, float principle){
  return principle * rate /100;
}

File: Week 2 PY/interest.py
def firstDecimal(num):
    return int(num%1*10)
#if you don't use firstDecimal, with or without the round function in simple, the str output takes the int and str's it

def interest(rate, principle):
    simple = round(rate/100 * principle, 2)
    output = "$" + str(int(simple)) + "." + \
        str(firstDecimal(simple)) + str(firstDecimal(simple*10))
    return output

rate = input("Enter the interest rate as a percentage: ")
principle = input("Enter the principle of the loan: ")
print("Your simple interest amount is " + interest(float(rate), float(principle)))